core/src/tangram.cpp

```diff
@@ -241,6 +247,23 @@ void render() {
     FrameInfo::draw(*m_view, *m_tileManager);
 
     while (Error::hadGlError("Tangram::render()")) {}
+
+
+    bool ready = !(m_view->changedOnLastUpdate() ||
+                   m_tileManager->hasTileSetChanged() ||
+                   m_tileManager->hasLoadingTiles() ||
+                   m_labels->needUpdate());
+    if (ready) {
+        for (auto& ease : m_eases) {
+            if (!ease.finished()) { ready = false; }
+        }
+    }
+
+    m_renderDoneCallback(ready);
```

Comment:
It's just the name that implies for me that this callback should be called at the end of each frame, but yes, for this
ViewCompleteCallback would probably be more appropriate (though it sounds more like it is only triggered when the state changes to complete, not for each frame that is complete)

I think it would be nicer to query the frame ready state in onDraw() instead
```
      if (viewCompleteListener != null) {
           if (nativeFrameRenderedWithAllTilesLoadedAndNoAnimationRunning()) {
               viewCompleteListener.onViewComplete();
           }
      }
```